integration/spark/src/main/scala/org/apache/carbondata/spark/rdd/NewCarbonDataLoadRDD.scala

```diff
@@ -135,7 +136,8 @@ class NewCarbonDataLoadRDD[K, V](
 
         val preFetch = CarbonProperties.getInstance().getProperty(CarbonCommonConstants
           .USE_PREFETCH_WHILE_LOADING, CarbonCommonConstants.USE_PREFETCH_WHILE_LOADING_DEFAULT)
-        carbonLoadModel.setPreFetch(preFetch.toBoolean)
+        carbonLoadModel.setPreFetch(Try(preFetch.toBoolean).getOrElse((CarbonCommonConstants
```

Comment:
remove unnecessary brackets for `(CarbonCommonConstants
          .USE_PREFETCH_WHILE_LOADING_DEFAULT).toBoolean`

Reply:
done